packages/core/src/components/ColorPalette/ColorDisplay.tsx

```diff
@@ -224,7 +224,7 @@ export function ColorDisplay({
             }
             return (
               <motion.div
-                key={variant.name}
+                key={`${colorName}-${variant.name}`}
```

Comment:
because the hsl affects all other colors

keys might be a red herring tho

Reply:
The key does not have the hex code in it anymore rather the name of the color name and variant name. Meaning like Purple-50, Purple-100 ... . It will only should the animation once due to display colors being memoized.